code/api/src/config/database.json

```diff
@@ -5,6 +5,13 @@
     "host": "127.0.0.1",
     "dialect": "postgres"
   },
+  "test": {
+    "password": null,
+    "database": "crate_test",
+    "host": "127.0.0.1",
+    "dialect": "postgres",
+    "seederStorage": "sequelize"
```

Comment:
We might want to remove this if we tear down every test. I can look into before all etc.